howso/direct/client.py

```diff
@@ -1130,6 +1149,7 @@ def copy_trainee(
         *,
         library_type: t.Optional[LibraryType] = None,
         resources: t.Optional[dict] = None,
```

Comment:
Lets change this Mapping to match the base class since it now gives some linting warnings

Reply:
fixed